x-pack/functionbeat/provider/aws/config.go

```diff
@@ -26,11 +26,11 @@ const maxMegabytes = 3008
 var DefaultLambdaConfig = &lambdaConfig{
 	MemorySize:  128 * 1024 * 1024,
 	Timeout:     time.Second * 3,
-	Concurrency: 0, // unreserve
+	Concurrency: 5,
 }
 
 type lambdaConfig struct {
-	Concurrency      int               `config:"concurrency" validate:"positive"`
+	Concurrency      int               `config:"concurrency" validate:"min=0,max=1000"`
```

Comment:
I'm wondering, is 1000 as hard maximum in AWS?

Reply:
It is indeed a weird setting, it is a hard limit per account of the total of running function of any kind.

IE: if you set this function to 5, It will run a maximum of 5 of your function.